pytket/qir/conversion/baseprofileqirgenerator.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019-2024 Quantinuum
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+import pyqir
+from pyqir import Value
+
+from pytket import Bit, Circuit, Qubit, wasm  # type: ignore
```

Comment:
Prefer `from pytket.wasm import WasmFileHandler` and use that directly; then we may not need this "type: ignore"?

Reply:
I have changed that, see [daaa907](https://github.com/CQCL/pytket-qir/pull/177/commits/daaa90774198cac867c27b97be051e87d1a32474)